app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -49,7 +45,8 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
     checkRedirectResult();
   }, [setFlowState]);
 
-  async function handleSubmit({ email, name }: FormValues) {
+  async function handleSubmit(values: Record<string, string>) {
+    const { email, name } = values as unknown as FormValues; // temp solution
```

Comment:
Is there a better solution than this temporary one? Why have you left this solution in here?

Reply:
There's another ticket to refactor this function. I used this temp solution so I can pass the type check.

Reply:
Fixed the temp solution issue to ensure best practices.